websurvey/lib/typewheels/responseStore.js

```diff
@@ -31,6 +31,11 @@ class ResponseStore {
       };
     }
   }
+
+  interpolate(field, qa) {
```

Comment:
An idea: if you return the whole field, instead of just the title, then later you can add interpolation of choices too, without breaking the API of this function!